apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx

```diff
@@ -1,39 +1,28 @@
-import React, { ReactNode, useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
 import remoteConfig from "@react-native-firebase/remote-config";
-import { DEFAULT_FEATURES } from "@ledgerhq/live-common/featureFlags/index";
-import { reduce, snakeCase } from "lodash";
-import { FeatureMap } from "@ledgerhq/types-live";
-
-export const formatToFirebaseFeatureId = (id: string) => {
-  return `feature_${snakeCase(id)}`;
-};
-
-// Firebase SDK treat JSON values as strings
-const formatDefaultFeatures = (config: FeatureMap) =>
-  reduce(
-    config,
-    (acc, feature, featureId) => ({
-      ...acc,
-      [formatToFirebaseFeatureId(featureId)]: JSON.stringify(feature),
-    }),
-    {},
-  );
-
-type Props = {
-  children?: ReactNode;
-};
+import { DEFAULT_FEATURES, formatDefaultFeatures } from "@ledgerhq/live-config/featureFlags/index";
+import type { FirebaseFeatureFlagsProviderProps as Props } from "@ledgerhq/live-config/featureFlags/index";
+import { LiveConfig } from "@ledgerhq/live-config/featureFlags/index";
 
 export const FirebaseRemoteConfigProvider = ({ children }: Props): JSX.Element | null => {
   const [loaded, setLoaded] = useState<boolean>(false);
 
   useEffect(() => {
     let unmounted = false;
-    const loadRemoteConfig = async () => {
+    const fetchConfig = async () => {
       try {
+        if (__DEV__) {
+          remoteConfig().setConfigSettings({ minimumFetchIntervalMillis: 0 });
```

Comment:
no, we can't do that in production. as mentioned in: https://firebase.google.com/docs/remote-config/get-started?platform=android#caching

"if you pushed your app out to thousands of test users with a very low minimum fetch interval, your app would probably hit this quota."

you can see the same logic in the existing code:
https://github.com/LedgerHQ/ledger-live/blob/b7d58b4cf3aa041b7b794d9af6f0b89bbc0df633/apps/ledger-live-mobile/src/components/FirebaseFeatureFlags.tsx#L78

Reply:
Agree that a value of 0 in prod isn't possible. Just to add some consideration, to me the doc isn't very clear on how much below 12h we can go, I think a value below 12h is possible, just without guarantee that all users will get the config with that frequency.

I suspect we could safely use something below 12h, but no idea how low we could go.

Reply:
Yeah depends on how many time on average a user close/open LL in a 12h range. Let's remember that the `0` value means that we fetch every time the user open Ledger instead of fetching on next opening if last fetch was less than 12 hours ago.
Do we have an idea of what our actual rate limit is ? Is it calculated per user or in total ?
We can try to lower the number to something like 4 hours and see how fast the rate limit progress